### ISSUE TITLE
Trouble with SHA-1 files

### ISSUE THREAD

**Issue by asymmetric** [NixOS/mvn2nix-maven-plugin#14]
I'm trying to package the [kframework](https://github.com/kframework/k) project, which [uses modules](#4), and I'm seeing the following error:

```

**Comment by jerith666** [comment]
Are you expecting only k-framework-related artifacts to reside in those repos?  The artifact it's failing on is a core maven artifact (`org.codehaus.plexus:plexus-utils`), which I'd usually expect to be obtained from maven central.

(Unless you're using a repo manager that does proxying, like nexus or artifactory, but given the S3 URLs I'm assuming that's not the case.)

**Comment by asymmetric** [comment]
@jerith666 you're correct in that it would be best practice to fetch it from maven central first, and that's what the [solution](https://github.com/kframework/k/commit/24fd6b984d1f1940d73e6a99dac7c13041284bb7) ended up being.

If you think there's nothing to be done on this tool's end, then we might as well close this issue.

My reasoning for opening it though is that maven built fine, whereas this tool errored on the missing `.sha1`, so I thought there could be something to fix.

**Comment by ttuegel** [comment]
@asymmetric Were you ultimately able to get K to build?

**Comment by asymmetric** [comment]
@ttuegel No, because of the problem you outlined in https://github.com/NixOS/mvn2nix-maven-plugin/issues/5#issuecomment-407763904.

**Comment by jerith666** [comment]
> My reasoning for opening it though is that maven built fine, whereas this tool errored on the missing .sha1, so I thought there could be something to fix.

Yeah, I'm not familiar enough with the details of the maven repo format or this tool to say where the real problem was ... :-/
